test/v1/service.go

```diff
@@ -182,12 +180,12 @@ func PatchService(t pkgTest.T, clients *test.Clients, service *v1.Service, fopt
 
 // UpdateServiceRouteSpec updates a service to use the route name in names.
 func UpdateServiceRouteSpec(t pkgTest.T, clients *test.Clients, names test.ResourceNames, rs v1.RouteSpec) (svc *v1.Service, err error) {
-	patches := []jsonpatch.JsonPatchOperation{{
```

Comment:
curious why this used an array before.. I think patch _does_ normally take an array of operations, but I guess the tests would be failing if it didn't also support a single op 🤔

Reply:
... actually, looks like some of the tests _are_ failing, wonder if that's why!

Reply:
Prolly, checking again, thanks!